transcom/mymove
# B-18193 API Fix Delete Weight Ticket

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a871909)

## Summary

Move query from handler to the service

### How to test

1. Access the application as a customer in the PPM Document upload stage
2. Click Upload PPM Documents
3. Fill out the required info and advance to the weight ticket submission screen
4. Fill out the required info and submit a weight ticket
5. Attempt to delete the weight ticket and confirm it deletes
